cli/src/main/java/com/devonfw/tools/ide/url/model/file/json/UrlStatus.java

```diff
@@ -48,4 +50,29 @@ public void setError(UrlStatusState error) {
 
     this.error = error;
   }
+
+  /**
+   * @return {@code true} if entirely empty, {@code false} otherwise.
+   */
+  public boolean checkEmpty() {
+
+    return (this.error == null) && (this.success == null);
+  }
+
+  public boolean checkStillUsed() {
```

Comment:
If you use a regular getter (`isStillUsed()`), then Jackson will write this property into JSON.